src/main/ui/src/components/Header/Header.jsx

```diff
@@ -3,6 +3,11 @@ import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import SearchBar from "../SearchBar/SearchBar";
 import StateDropdown from "./StateDropdown";
+import Button from 'react-bootstrap/Button';
+
```

Comment:
remove all this space please. Doesn't look as if you install the linter like I asked a while ago. I'll DM you the link to setup